doc/user/bgp.rst

```diff
@@ -2751,7 +2751,6 @@ route with gateway IP.
 1. CLI to add gateway IP while generating EVPN type-5 route from a BGP IPv4/IPv6
 prefix:
 
-.. index:: advertise <ipv4|ipv6> unicast [gateway-ip]
 .. clicmd:: [no] advertise <ipv4|ipv6> unicast [gateway-ip]
```

Comment:
Do we still demand for `[no]` (negative form)?

Reply:
Right, this should also be fixed.